dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/logging/LogUtils.java

```diff
@@ -165,9 +166,8 @@ public static void setLoggerClass(Class<? extends AbstractDelegatingLogger> cls)
      * @return an appropriate Logger
      */
     public static Logger getLogger(Class<?> cls) {
-        //Liberty Change for CXF Begain
-        return createLogger(cls, null, cls.getName() + getClassLoader(cls));
-        //Liberty Change for CXF End
+	Logger logger = createLogger(cls, null, cls.getName() + getClassLoader(cls));
```

Comment:
I'm fine with the added trace for this file, but will you make sure that the trace injection is turned off for this class in the `bnd.bnd`?

Reply:
I removed all trace from LogUtils.java with the commit  "Remove trace from LogUtils.java ".

Reply:
I already removed all trace from this class.